test/jdk/java/net/URLPermission/OpenURL.java

```diff
@@ -25,33 +25,76 @@
  * @test
  * @bug 8029354
  * @library /test/lib
- * @run main/othervm -Djava.security.manager=allow OpenURL
+ * @run main/othervm OpenURL
  */
 
```

Comment:
Do we need to keep this test at all? Or if keeping it, does it need the HTTP server simulation, since the bug only relates to the URLPermission instantiation? The test could be reduced to just the URL creation followed by the URLPermission.

Reply:
It could - but technically openConnection / getInputStream could still throw if there was an issue with the provided URL. The difference here is that with a SecurityManager the connection would be rejected with a SecurityException before the connection was made. Without a security manager, the connection will go through, so you need the server simulation (can't rely on getting IOException assuming nobody would be listening).